Orange/widgets/data/owimpute.py

```diff
@@ -94,8 +95,16 @@ def initStyleOption(self, option, index):
             fmt = state.method.format
             text = fmt.format(var=var, params=state.params,
                               **state.method._asdict())
+
+            if error_shown and var in not_compatible_with_learner \
```

Comment:
PEP8 says to prefer `(...)` for values spanning multiple lines (i.e. wrapping it all in parentheses).